vendor/init.bat

```diff
@@ -6,6 +6,9 @@
 :: !!! THIS FILE IS OVERWRITTEN WHEN CMDER IS UPDATED
 :: !!! Use "%CMDER_ROOT%\config\user-profile.cmd" to add your own startup commands
 
+:: Set to > 0 for verbose output to aid in debugging.
+set verbose-output=0
```

Comment:
Done.

Add 'set verbose-output=1 && ' before 'cmd /k' in cmder task definition.